alacritty_terminal/src/grid/row.rs

```diff
@@ -80,11 +84,17 @@ impl<T: Copy> Row<T> {
     }
 
     /// Resets contents to the contents of `other`
-    pub fn reset(&mut self, other: &T) {
-        for item in &mut self.inner[..] {
-            *item = *other;
+    pub fn reset(&mut self, template: &T)
```

Comment:
Nice, `other` was confusing.

Reply:
Yeah, we've used `template` everywhere else, so being consistent here makes a lot more sense.